### ISSUE TITLE
assigning an IDispatch* (VT_DISPATCH type)  to _variant_t

### ISSUE THREAD

**Issue by chund** [MicrosoftDocs/cpp-docs#3027]
This could be improved:
operator=( pDispSrc ) Assigns a VT_DISPATCH object to a _variant_t object.
operator=( pIUnknownSrc ) Assigns a VT_UNKNOWN object to a _variant_t object.

actually you do not assign VT_DISPATCH or  VT_UNKNOWN but a IDispatch* or IUnknown*
operator=( pDispSrc ) Assigns a  IDispatch* object to a _variant_t object.
operator=( pIUnknownSrc ) Assigns a IUnknown* object to a _variant_t object.

furthermore you could add, that _variant_t will take care of calling AddRef() and Release() on these objects.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 11ac5410-e21b-e1d9-db46-52d1ca271512
* Version Independent ID: 020d2cb6-144e-460b-ecc2-eef166713832
* Content: [_variant_t::operator =](https://docs.microsoft.com/en-us/cpp/cpp/variant-t-operator-equal?view=msvc-160)
* Content Source: [docs/cpp/variant-t-operator-equal.md](https://github.com/Microsoft/cpp-docs/blob/master/docs/cpp/variant-t-operator-equal.md)
* Product: **visual-cpp**
* Technology: **cpp-language**
* GitHub Login: @corob-msft
* Microsoft Alias: **corob**

**Comment by corob-msft** [comment]
@chund 
I don't know if anyone has reported being confused by this documentation, but you're certainly not wrong. Would you like to submit a PR with your proposed changes? Don't worry about perfect style or grammar, we'll fix problems like that for you.

**Comment by MUzairS15** [comment]
Can you tell me from where to start as i am new to open source?

**Comment by corob-msft** [comment]
@MUzairS15 
Good question, thanks for asking. I should probably include this in everything tagged "up for grabs":

We have a [Contributor guide](https://docs.microsoft.com/contribute/) available for Docs. The contributor guide walks you through the steps for a simple change: First, use the Edit control (a pencil icon) at the top of an article to open the article for editing in the GitHub web interface. Make your changes in the web editor, using [markdown](https://github.github.com/gfm/) for formatting control. Then use the edit boxes below the article text editor to describe your changes and make a pull request. You may need to agree to a contributor license to complete the pull request, if it's more than a trivial change. Once the PR is created, we'll take it from there to edit and approve your PR, and then you'll get credited as a contributor. 

It's also possible to fork the whole cpp-docs repo to make more extensive changes that involve more than one article, but I don't recommend that for beginners. 

Thanks for the interest!

**Comment by MUzairS15** [comment]
May I know why this is still open?

**Comment by corob-msft** [comment]
@MUzairS15 
To let us know that a PR fixes a particular issue, you can include a link to the issue in the PR comment. The simplest form of link looks like `#3027`, or in a slightly longer form that works across GitHub repos, `MicrosoftDocs/cpp-docs/issues/3027`. So, for example, you might include a comment like:

```
Update to description for IDispatch* and IUnknown* arguments.
Fixes #3027.
```

With this link in the comment, GitHub automatically marks the issue closed when the PR is merged.
